### PR TITLE
Fixed x-editable datepicker

### PR DESCRIPTION
**Pull request by core23**
See https://github.com/sonata-project/SonataAdminBundle/pull/3353

### CODE REVIEW

File: Resources/public/css/styles.css [hunk 1]
```diff
@@ -333,3 +333,11 @@ div.select2-container {
...
     margin: 0 !important;
 }
+
+.editable-container .prev:before {
+    content: "\2190 ";
+}
+
+.editable-container .next:before {
+    content: "\2192 ";
+}
```
Comment (OskarStark) [inline Resources/public/css/styles.css]: no new line @core23

**Comment by core23** [comment]
Removed the newlines and put the classes under the other x-ediable declarations.

**Comment by Soullivaneuh** [comment]
Thank you @core23

**Comment by OskarStark** [comment]
:+1:

**Pull request merged**